src/modalities/models/gpt2/gpt2_model.py

```diff
@@ -519,7 +518,9 @@ def forward_impl(self, inputs: Dict[str, torch.Tensor]) -> Dict[str, torch.Tenso
         input_ids = inputs[self.sample_key]
         device = input_ids.device
         b, t = input_ids.size()  # batch size, sequence length
```

Comment:
```suggestion
        _, t = input_ids.size()  # batch size, sequence length
```
batch size `b` is not used